launch/twist_mux_launch.py

```diff
@@ -48,12 +48,17 @@ def generate_launch_description():
             'cmd_vel_out',
             default_value='twist_mux/cmd_vel',
             description='cmd vel output topic'),
+        DeclareLaunchArgument(
+            'use_sim_time',
+            default_value='True',
```

Comment:
Can we have it off by default to not change default behaviour?
```suggestion
            default_value='False',
```